### PR TITLE
Portfolio Accessibility

### PR DESCRIPTION
**Pull request by HFG43**
Hi there, we are sending this PR for our code revision! 📝 
We have check all page & items to ensure we comply with accessibility standard requirements.
We have prepared this project by doing mob programming.

The result of the check and changes was the following:

- Page Title: changed, we did it more descriptive (https://github.com/CarlosIgreda/Portfolio-setup-and-mobile-first/compare/main...accessibility#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L9-R9)
- Images: corrected Alt, giving a more descriptive alternative text: Snapshot Images, logos, item. Hand icon set to null (https://github.com/CarlosIgreda/Portfolio-setup-and-mobile-first/compare/main...accessibility#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L9-R337)
- Headings: H1 – H2 - Changed h4 as subheading in cards, and replaced it with a h3, to respect hierarchy (https://github.com/CarlosIgreda/Portfolio-setup-and-mobile-first/compare/main...accessibility#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L86-R240)
- Color-contrasts: Future Features to be included 
- Resize: correct background size & position in "Contact Form" image (https://github.com/CarlosIgreda/Portfolio-setup-and-mobile-first/compare/main...accessibility#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1L639-R660)
- Interaction: Add labels in the form for voice-over reader using “text-indent:-999em” (https://github.com/CarlosIgreda/Portfolio-setup-and-mobile-first/compare/main...accessibility#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R355-R361) & (https://github.com/CarlosIgreda/Portfolio-setup-and-mobile-first/compare/main...accessibility#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1R752-R757)
- Moving:  not an issue
- Multimedia: not an issue
- Basic structure of Page: no issues. Check done using Web Developer tool

Thank you in advance for your feedback! 🚀

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
                         My Logo
                     </a>
                     <a href="#">
-                        <img id="hamburger" src="./images/hamburger.svg" alt="menu-icon">
+                        <img id="hamburger" src="./images/hamburger.svg" alt="A menu icon">
                     </a>
```
Comment (Shedrack-Sunday) [inline index.html:26]: - [x] As part of the interactivity part of web accessibility, It's possible to navigate through the page, focus on its elements, and select and navigate through its links. Right now, your hamburger is dormant with no interaction. For a better user experience and to maintain industry standards for accessibility, kindly adjust this to keep up with your brillant job so far 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @HFG43 @CarlosIgreda @DamianDib9      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper PR summary 
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required a minor adjustments to the hamburger 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (CarlosIgreda) [reply index.html:26]: Hi @Shedrack-Sunday, could you give us an example of that mentioned interaction on the hamburger icon? I'd really appreciate it in order to complete this task. Thanks in advance buddy.

Comment (CarlosIgreda) [reply index.html:26]: Done it @Shedrack-Sunday, I've added an interaction to the hamburger icon.

**Review by didierganthier — APPROVED** [review]
Hi Team,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Comment by CarlosIgreda** [comment]
Great @didierganthier, thanks for your review 🤓

**Pull request merged**